### DISCUSSION TITLE
Torch Optimizations --> also in Windows?

### DISCUSSION THREAD

**Discussion by cibernicola** [vladmandic/sdnext#371] [Q&A]
Hello, 
First of all, from what I'm seeing, congratulations, I can see the desire to move forward and do it in a coherent way.

The doubt is relatively simple, the optimizations that are talked about here https://github.com/vladmandic/automatic/wiki/Torch-Optimizations, are also applicable in Windows? If so, perhaps it would be necessary to add the steps in the Wiki.

Thank you!

**Comment by vladmandic** [comment]
thanks.
primary goal is to make setup intelligent enough to handle 99% of cases and then document in wiki with whatever is left.
but yes, wiki update is pending for sure.

**Reply by cibernicola** [reply]
If I can help testing or whatever... Here I am :)
